src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -194,7 +194,15 @@ function MoneyRequestPreviewContent({
         }
 
         if (shouldShowRBR && transaction) {
-            const violations = TransactionUtils.getTransactionViolations(transaction.transactionID, transactionViolations);
+            const violations = TransactionUtils.getTransactionViolations(transaction.transactionID, transactionViolations)?.sort((a, b) => {
```

Comment:
https://github.com/Expensify/App/blob/39463bdccefd9017b3f4beea889e57e58ad5644e/src/components/SettlementButton.tsx#L224

Should we use this way? I think it will be more concise and comprehensible

Reply:
@daledah Friendly bump!